### ISSUE TITLE
proposal: accordion style preview

### ISSUE THREAD

**Issue by Max104t** [gitnew2018/My-OpenPeriscope#30]
how do you feel about integrating the preview inlined inside the broadcast card.. it may expand and collapse as an accordion.
![accordion-preview](https://user-images.githubusercontent.com/49573478/62416673-0ad6ae80-b5f4-11e9-8d1e-25eee2537b07.png)

**Comment by gitnew2018** [comment]
Not sure, I don't really see any benefits.
Maybe other users will like it.

**Comment by kewalsk** [comment]
There could be one benefit if you are able to integrate private broadcasts this way. The standard view is not working as the browser don't have necessary cookies.

**Comment by gitnew2018** [comment]
It is only for preview (screenlist), not actual video.
It might be possible to play privates in new windows. There is a file inject.js that can do "things" on the opened website.

**Comment by gitnew2018** [comment]
@kewalsk you can actually watch private broadcasts, you just have to log in when the broadcast window opens.

**Comment by kewalsk** [comment]
Yes, looks like some solution, but this way I have to login twice. Maybe setting some cookies could make something like auto-login. The other problem I have is that the broadcasts are not playing in chrome at all in my system and I don't know why (firefox works fine).

**Comment by gitnew2018** [comment]
Chrome? or chromium because chromium(including nwjs) doesn't include proprietary codecs. This one works for me https://github.com/iteufel/nwjs-ffmpeg-prebuilt/releases.

**Comment by kewalsk** [comment]
Yes, chromium. The chrome browser works fine. Thanks for the solution - replacing ffmpeg.dll solved the  problem even with the newest nw.js version.
